Mathlib/GroupTheory/OrderOfElement.lean

```diff
@@ -80,11 +80,29 @@ theorem not_isOfFinOrder_of_injective_pow {x : G} (h : Injective fun n : ℕ =>
   rw [h hnx] at hn_pos
   exact irrefl 0 hn_pos
 
+/-- 1 is of finite order in any monoid. -/
+@[to_additive (attr := simp) "0 is of finite order in any additive monoid."]
+theorem isOfFinOrder_one : IsOfFinOrder (1 : G) :=
```

Comment:
Could you take the opportunity to rename
```suggestion
theorem IsOfFinOrder.one : IsOfFinOrder (1 : G) :=
```
for anonymous dot notation?

Reply:
Done (+added deprecated aliases).